druid-shell/src/platform/x11/window.rs

```diff
@@ -693,16 +617,11 @@ impl Window {
         &self,
         _destroy_notify: &xcb::DestroyNotifyEvent,
     ) -> Result<(), Error> {
-        match self.handler.try_borrow_mut() {
-            Ok(mut handler) => {
-                handler.destroy();
-                Ok(())
-            }
-            Err(err) => Err(Error::BorrowError(format!(
-                "Window::handle_destroy_notify handle: {}",
-                err
-            ))),
-        }
+        self.handler
+            .try_borrow_mut()
+            .context("Window::handle_destroy_notify")?
```

Comment:
Lets keep the variable name in the context message. (`handler` instead of `handle` though)